RPGManager_development/test/unit_test/character_sheet_unit_tests/ArmorsUnitTests.java

```diff
@@ -0,0 +1,145 @@
+package unit_test.character_sheet_unit_tests;

```

Comment:
Átírtam, tökéletesen működik. Viszont, a default armor rating az 0, mivel Integer.class ként van beállítva. Ez szerinted így maradjon, legyen belőle egy class, vagy van valami ötleted hogyan lehetne  átálitani az értékét 4-re.